src/confusion_matrix.jl

```diff
@@ -7,6 +7,20 @@ struct ConfusionMatrix{T<:Real}
     fn::T   # (incorrect) negative prediction when target is positive
 end
 
+function show(io::IO, cm::ConfusionMatrix)
```

Comment:
Please change to

`function show(io::IO, ::MIME"text/plain", cm::ConfusionMatrix)`

so that

```
julia> [ConfusionMatrix(2, 2, 1, 1, 1, 1), ConfusionMatrix(2, 2, 1, 1, 1, 1)]
```

works fine.

Reply:
Check ^_^